@re-/model/demos/generateCode/README.md

```diff
@@ -0,0 +1,12 @@
+All files are copied to mappedInfo[files] regardless
```

Comment:
This looks like a great strategy overall for solving the problem. Really excited to start using it 😻